config.py

```diff
@@ -31,6 +31,7 @@
     'id': '{}-tf-mobilenet'.format(MODEL_NAME.lower()),
     'name': '{} TensorFlow Model'.format(MODEL_NAME),
     'description': '{} TensorFlow model trained on MobileNet'.format(MODEL_NAME),
-    'type': 'object_detection',
+    'type': 'Object Detection',
```

Comment:
I believe the type is meant to match the match the primary model tag, which should be a slug (am I correct in this @MLnick ?)

Reply:
Ideally yes it would be matching the slug. However as @splovyt says it is impossible to really enforce in practice apart from "by convention". It also then means a user needs to go look up the full name for a given slug (though the slugs are similar to the full name and generally explanatory, it may not always be the case).

I'm not hugely leaning either way. Perhaps it is a bit more intuitive to use the full name here and have that match the model's primary tag on the MAX site, by convention.